### ISSUE TITLE
Review release 4.5.3

### ISSUE THREAD

**Issue by phpgeekfr** [blockframes/blockframes#8990]
To test :
- [ ] check if new library updates are ok https://github.com/blockframes/blockframes/pull/8974 (bruce)

**Comment by cams-goussale** [comment]
- [ ] I thought we were removing the page number in the url? It's still here (https://github.com/blockframes/blockframes/issues/8893#issuecomment-1245161948)
- [ ] When loading all movies on festival app, this is still not showing (https://github.com/blockframes/blockframes/issues/8893#issuecomment-1245355565)
![image](https://user-images.githubusercontent.com/93206363/198541030-eb645034-0063-4472-9294-c1ec3509ce5b.png)

**Comment by phpgeekfr** [comment]
> I thought we were removing the page number in the url? It's still here on festival

Afterall, this was not the good solution. I made it work by keeping the page number in url but resetting when search filters are changed

> On the contrary there is no page number in the url for catalog, so as soon as user has loaded all titles once, then they're always all showing.

You mean for example if I load 200 titles for specific search filters, then change filters, I still have 200 titles displayed for this new research instead of 50 ?

> When loading all movies on festival app, this is still not showing on festival

Indeed, looking at it

>The page parameter is still in the saved research url

 It is removed from saved search but on load, it is always set to zero

> I realized that the saved research isn't stored anywhere in the db. So if user logs out (or clear cache or maybe after a release), user loses its saved research.

Indeed, I did not noticed that login out would clear the saved search.. During a release it should not.
Saving it in DB should not be that long. Do you want me to have a look at it for this release ?

**Comment by cams-goussale** [comment]
> You mean for example if I load 200 titles for specific search filters, then change filters, I still have 200 titles displayed for this new research instead of 50 ?

No, I mean if I load 200 titles for a specific search filters, then change filters, and then go back to initial search, there is still 200 titles displayed. Whereas if I do the same on festival app, it will load only 50 ; because page number is reset after each filter change. (let me know if not clear ^^)

It seems to be the expected behavior (based on your explanations). Let's just keep in mind that the behaviors are different for each app.

> Indeed, I did not noticed that login out would clear the saved search.. During a release it should not.
Saving it in DB should not be that long. Do you want me to have a look at it for this release ?

Yes, if not too long.

**Comment by phpgeekfr** [comment]
> When loading all movies on festival app, this is still not showing on festival

Ok it was a limitation config on algolia dashboard.
![image.png](https://images.zenhubusercontent.com/5c6e955fdecd2e3c69b6c1d2/df470bdb-e647-4839-b78d-d94074519553)

This setting can be configured when deploying a new release but only if we have a db migration programmed (algolia does not update during a release unless there is a migration).
- [ ] So I will update it manually for staging and production for this release and this will be set on next release with migration 

> On the contrary there is no page number in the url for catalog, so as soon as user has loaded all titles once, then they're always all showing.

If you don't mind, as this is not blocking and just a different behavior than other apps, I would like to do this in #8894

**Comment by phpgeekfr** [comment]
> So I will update it manually for staging and production for this release and this will be set on next release with migration

Updated manually to 2000 results for `staging_festival_movies` and `prod_festival_movies` other indexes does not need it for now
